legacy/src/app/partials/node-details.html

```diff
@@ -673,7 +678,8 @@ <h1 class="page-header__title">
               class="p-tabs__link"
               data-ng-if="node.testing_status.pending > 0 || node.testing_status.running > 0 || node.testing_status.passed > 0 || node.testing_status.failed > 0"
               data-ng-class="{ 'is-active': section.area === 'testing'}"
-              data-ng-click="openSection('testing')"
+              data-ng-click="isController || isDevice ? openSection('testing') : navigateToNew('/machine/' + node.system_id + '/tests')"
+              href="{$ isController || isDevice ? undefined : newURLBase + '/machine/' + node.system_id + '/tests' $}"
```

Comment:
Should we change this to be `testing` to match the old urls?

Reply:
Yeah that's a good idea actually - probably shouldn't change the url just for the sake of it